python/ray/autoscaler/commands.py

```diff
@@ -344,9 +344,41 @@ def remaining_nodes():
 
             return head + workers
 
+        def run_docker_stop(node, container_name):
+            try:
+                updater = NodeUpdaterThread(
+                    node_id=node,
+                    provider_config=config["provider"],
+                    provider=provider,
+                    auth_config=config["auth"],
+                    cluster_name=config["cluster_name"],
+                    file_mounts=config["file_mounts"],
+                    initialization_commands=[],
+                    setup_commands=[],
+                    ray_start_commands=[],
+                    runtime_hash="",
+                    file_mounts_contents_hash="",
+                    is_head_node=False,
+                    docker_config=config.get("docker"))
+                _exec(
+                    updater,
+                    f"docker stop {container_name}",
+                    False,
+                    False,
+                    run_env="host")
+            except Exception:
+                cli_logger.warning(f"Docker stop failed on {node}")
+                cli_logger.old_warning(f"Docker stopped on {node}")
```

Comment:
I think old_xxx logging requires `logger` to be passed in.

Reply:
Also, don't these two errors say opposite things ?

Reply:
Yep! Fixed!